Fix to resolve track propertly in Library.lookup/

jcass77 opened the pull request:
Fixes https://github.com/mopidy/mopidy-youtube/issues/21 again: was not resolving URIs of type ```youtube:video``` properly as they need to be passed through ```resolve_track``` first in order to extract the video ID that can be used with ```pafy.new()```.

The actual fix is contained on line https://github.com/mopidy/mopidy-youtube/blob/6927f51f5055fb92c36cfbbe269b16abd47ff536/mopidy_youtube/backend.py#L157

jodal commented on mopidy_youtube/backend.py:
```diff
@@ -49,7 +52,7 @@ def resolve_url(url, stream=False):
     try:
         video = pafy.new(url)
         if not stream:
-            uri = 'youtube:video/%s.%s' % (
```
If I understand it correctly, the intention of using slash in the URI here was to help a number of clients to show just the last part of the URI as the name, e.g. "title foo bar baz.the_id_we_actually_use". Changing the `/` to `:` will break that.

jodal commented:
Bonus points if you update the changelog ;-)

jcass77 commented:
Ok, I've fixed the separator, added a test case for issue https://github.com/mopidy/mopidy-youtube/issues/21, and updated the changelog.

I  also changed the URI of the youtube video that is used in the tests as the current one appears to be geo-locked?

jodal commented on README.rst:
```diff
@@ -77,6 +77,11 @@ Project resources
 Changelog
 =========
 
+v2.0.2 (2015-11-30)
```
Next time, just write "UNRELEASED" in the parens.

jodal commented on mopidy_youtube/__init__.py:
```diff
@@ -6,7 +6,7 @@
 
 
-__version__ = '2.0.1'
+__version__ = '2.0.2'
```
Next time, just leave the version as is. We'll bump it when we release.

jodal commented on mopidy_youtube/backend.py:
```diff
@@ -49,7 +52,7 @@ def resolve_url(url, stream=False):
         video = pafy.new(url)
         if not stream:
-            uri = 'youtube:video/%s.%s' % (
+            uri = video_uri_prefix + '/%s.%s' % (
```
`'%s/%s.%s' % (video_uri_prefix, safe_url(video.title), video.videoid)` might be better than mixing concatenation and string formatting.

jodal commented:
Due to my own guilt of leaving this on the floor since November I fixed my own comments and merged.

Thanks for your contribution and the patience! :-)
